<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePageStatus</class>
 <widget class="QWidget" name="HomePageStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_6">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>50</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU使用率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_8">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>50</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="memory_space">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>182</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0/0(MB)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="load_status_info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>182</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>运行流畅</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="memory_status" native="true">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>77</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="disk_status" native="true">
   <property name="geometry">
    <rect>
     <x>617</x>
     <y>77</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="line1_widget" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>43</y>
     <width>730</width>
     <height>1</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>4</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#808080</string>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="disk_space">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>182</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0G/0G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="cpu_status" native="true">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>77</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_5">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="load_status" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>77</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_7">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>50</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>内存使用率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="cpu_core_num">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>182</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0核心</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>负载状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dtk::Widget::DLabel</class>
   <extends>QLabel</extends>
   <header>DLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
